.changes/1.82.0.json

```diff
@@ -0,0 +1,22 @@
+{
+	"date": "2023-07-26",
```

Comment:
This file (and others) is already on master. Not sure what's going on here, maybe a github bug

Reply:
looks like it went away? Not sure if you did something or if git fixed it.